packaged/lisk.sh

```diff
@@ -117,6 +118,13 @@ download_blockchain() {
 	if [ "$DB_DOWNLOAD" = "Y" ]; then
 		rm -f "$DB_SNAPSHOT"
 		if [ "$BLOCKCHAIN_URL" = "" ]; then
+			# performing network check here
```

Comment:
Should we call `download_blockchain` at all in this case? (I can see there is already a check for `DB_DOWNLOAD` and quite a bit of logic in there but does it make sense?)

Reply:
As it is now, it leaves open the option to download the snapshot if the url is specified, irrespective of network